### PR TITLE
HTML Basics

### PR DESCRIPTION
**Pull request by asmockler**
Completes some of #6 
- Added an HTML Basics tutorial
- Added a basic HTML example
- Added a basic CSS example

cc: @danmo91

### CODE REVIEW

File: html-css-bootstrap/examples/html-basics/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Basic HTML example
+
+This directory contains 2 files: this readme and a file called `index.html`. `index.html` is a basic example of what an HTML document looks like. To view it in a browser, either copy the contents into a file on your local machine called `index.html` or right click the link to the file and choose `Save Link As`. You can open the file simply by double clicking.
```
Comment (danmo91) [inline html-css-bootstrap/examples/html-basics/README.md:3]: Very helpful onboarding instructions 💯

File: html-css-bootstrap/tutorials/html-basics.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
+# HTML Basics
+
+## Contents
+- [Todo](#todo)
```
Comment (danmo91) [inline html-css-bootstrap/tutorials/html-basics.md]: - [What is a Web Page?](#what-is-a-web-page)
- [HTML Elements](#html-elements)
- [HTML Structure](#html-structure)
- [Special Characters](#special-characters)
- [More Information](#more-information)

Comment (asmockler) [reply html-css-bootstrap/tutorials/html-basics.md]: Oops, thank you haha

File: html-css-bootstrap/tutorials/html-basics.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+
+### What is a Web Page?
+
+The Internet is based on a text-based request/response protocol called **HTTP** (Hyper Text Transfer Protocol). HTTP allows us to send text across the internet; **HTML** (Hyper Text Markup Language) is just a special kind of text that a computer uses to format web pages in a special way. When a computer receives text in a `.html` or `.htm` file, it knows to display it in a certain way.
+
+HTML files can be created in any text editor such as NotePad, NotePad++, Atom, SublimeText, Visual Studio, and many others.
```
Comment (danmo91) [inline html-css-bootstrap/tutorials/html-basics.md]: Do we really want to suggest NotePad or NotePad++???

File: html-css-bootstrap/tutorials/html-basics.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+
+### More attributes
+
+While many tags have attributes that perform special tasks, there are some attributes that are valid for almost any html element. The two most important of these attributes are `class` and `id`.
+
+* `class`, which allows you to label elements to be selected by CSS or JavaScript
+* `id`, which performs much like the `class` attribute except that ids must be unique on each page
+
+These will be covered further in the CSS Basics tutorial.
```
Comment (danmo91) [inline html-css-bootstrap/tutorials/html-basics.md]: could link to [CSS Basics Tutorial](https://github.com/byu-is-403/syllabus/blob/html-basics/html-css-bootstrap/tutorials/css-basics.md)

**Comment by danmo91** [comment]
this is where we came from:

bad

👏  THIS 👏  IS 👏  WHAT 👏  YOU 👏  MADE 👏 

good

**Pull request merged**